docs/guides/cluster-to-advanced-cluster-migration-guide.md

```diff
@@ -0,0 +1,175 @@
+
+Key differences in the configuration:
+
+1. Replication Spec Configuration: Supports different node types (electable, analytics, read_only) where hardware configuration can differ between node types.
+2. Provider Settings: Moved from the top level to the replication spec allowing you to create multi-cloud clusters.
+3. Auto Scaling: Moved from the top level to the replication spec allowing you to scale replication specs individually.
+4. Backup Configuration: Renamed from `cloud_backup` to `backup_enabled`.
+5. See also ["Migration to new sharding schema and leveraging Independent Shard Scaling"](advanced-cluster-new-sharding-schema.md#migration-sharded)
+
+### Example1: Old Configuration
+
+```terraform
+resource "mongodbatlas_cluster" "this" {
+    project_id                   = var.project_id
+    name                         = "legacy-cluster"
+    cluster_type                 = "REPLICASET"
+
+    # Provider Settings "block"
+    provider_instance_size_name = "M10" # 1 Replication Spec Configuration
+    provider_name               = "AWS" # 2 Provider Settings
+    
+    auto_scaling_disk_gb_enabled = true # 3 Auto Scaling
+    cloud_backup                 = true # 4 Backup Configuration
+
+    replication_specs {
+     num_shards = 1
+     regions_config {
+       region_name     = "US_EAST_1"
+       priority        = 7
+       electable_nodes = 3 # 1 Replication Spec Configuration
+       analytics_nodes = 1 # 1 Replication Spec Configuration
+       read_only_nodes = 0 # 1 Replication Spec Configuration
+       }
+     }
+
+    }
+```
+
+### Example2: New Configuration
+
+```terraform
+    resource "mongodbatlas_advanced_cluster" "this" {
+    project_id             = var.project_id
+    name                   = "advanced-cluster"
+    cluster_type           = "REPLICASET"
+    backup_enabled         = true # 4 Backup Configuration
+
+    # Replication specs
+    replication_specs {
+      auto_scaling { # 3 Auto Scaling
+        disk_gb_enabled = true
+      }
+      region_configs {
+        region_name   = "US_EAST_1"
+        priority    = 7
+        provider_name = "AWS" # 2 Provider Settings
+        
+        electable_specs { # 1 Replication Spec Configuration
+            instance_size = "M10"
+            node_count    = 3
+        }
+        analytics_specs { # 1 Replication Spec Configuration
+            instance_size = "M10"
+            node_count    = 1
+        }
+      }
+    }
+```
+
+### Output Changes
+
+- container_id:
+  - before: `mongodbatlas_cluster.this.replication_specs[0].container_id` was a flat string, e.g., `669644ae01bf814e3d25b963`
+  - after: `mongodbatlas_advanced_cluster.this.replication_specs[0].container_id` is a map, e.g., `{"AWS:US_EAST_1": "669644ae01bf814e3d25b963"}`
+
+## How to Change
+Before doing any migration it is recommended to make a backup of your Terraform state files. (ADD_LINK_HERE)
+
+### Method 1: `terraform plan -generate-config-out=cluster.tf`
```

Comment:
Not sure whether or not to enumerate different methods.
Once we have built more tooling around this.

Reply:
I agree that it might be less clear to format the one option as a list. I'd suggest updating to a list format after additional methods have been developed / released.

Reply:
Agree. Updated